_config.yml

```diff
@@ -145,13 +145,13 @@ authors:
     about: "Here's a little about emmtm"
     
   EnglandHam:
-    name: EnglandHam
+    name: Soojin Park
```

Comment:
I believe this can stay EnglandHam, I think your posts weren't showing up because the names you used in your individual postings. I'm not sure where exactly, you might want to talk to Elliott about it. I tend to only use my handle unless I'm customizing a post title.